### PR TITLE
PJ-1435 Used refresh token doesn't need to be revoked

### PR DESCRIPTION
**Pull request by imadex**

### CODE REVIEW

File: storage.go [hunk 1]
```diff
@@ -53,7 +53,4 @@ type Storage interface {
 	// AuthorizeData and AccessData DON'T NEED to be loaded if not easily available.
 	// Optionally can return error if expired.
 	LoadRefresh(token string) (*AccessData, error)
-
-	// RemoveRefresh revokes or deletes refresh AccessData.
-	RemoveRefresh(token string) error
```
Comment (anggorodewanto) [inline storage.go]: we don't need to modify the interface

File: access.go [hunk 2]
```diff
@@ -599,14 +599,6 @@ func (s *Server) FinishAccessRequest(w *Response, r *http.Request, ar *AccessReq
 			w.Storage.RemoveAuthorize(ret.AuthorizeData.Code)
 		}
 
-		// remove previous access token
-		if ret.AccessData != nil && !s.Config.RetainTokenAfterRefresh {
-			if ret.AccessData.RefreshToken != "" {
-				w.Storage.RemoveRefresh(ret.AccessData.RefreshToken)
-			}
-			w.Storage.RemoveAccess(ret.AccessData.AccessToken)
```
Comment (anggorodewanto) [inline access.go]: we still want to revoke the access token

File: storage.go (hunk 1 above)
Comment (imadex) [reply storage.go]: I have fixed it in this commit be9c164

File: access.go (hunk 2 above)
Comment (imadex) [reply access.go]: I have fixed it in this commit be9c164

**Comment by anggorodewanto** [comment]
LGTM

**Pull request merged**